templates/groups.sql.erb

```diff
@@ -0,0 +1,7 @@
+CREATE table IF NOT EXISTS groups (
```

Comment:
The export of the generated groups table looks like this:
```
CREATE TABLE groups (
    id integer NOT NULL,
    group_id integer,
    name text,
    created_at timestamp with time zone,
    updated_at timestamp with time zone,
    deleted_at timestamp with time zone
);

ALTER TABLE ONLY groups ALTER COLUMN id SET DEFAULT nextval('groups_id_seq'::regclass);
ALTER TABLE ONLY groups
    ADD CONSTRAINT groups_pkey PRIMARY KEY (id);
CREATE INDEX idx_groups_deleted_at ON groups USING btree (deleted_at);
CREATE UNIQUE INDEX idx_per_group ON groups USING btree (group_id, name);
```
Additionally it seems to me group_id is not needed as it's repeated with id, so probably [this](https://github.com/ErnestIO/group-store/blob/master/entity.go#L18) column is not needed?